command/server.go

```diff
@@ -73,6 +75,38 @@ func (c *ServerCommand) Run(args []string) int {
 		return 1
 	}
 
+	// Create a logger. We wrap it in a gated writer so that it doesn't
+	// start logging too early.
+	logGate := &gatedwriter.Writer{Writer: colorable.NewColorable(os.Stderr)}
+	var level int
+	switch logLevel {
+	case "trace":
+		level = log.LevelTrace
+	case "debug":
+		level = log.LevelDebug
+	case "info":
+		level = log.LevelInfo
+	case "notice":
+		level = log.LevelNotice
+	case "warn":
+		level = log.LevelWarn
+	case "err":
+		level = log.LevelError
+	default:
+		c.Ui.Output(fmt.Sprintf("Unknown log level %s", logLevel))
+		return 1
+	}
+	switch os.Getenv("LOGXI_FORMAT") {
```

Comment:
Right now this isn't really exposed, but the nice thing about leaving
it as-is is that people can use normal logxi formatters too.